File: fund/java/pacote/ex03/source/rh/TestarPessoa.java
package rh;

public class TestarPessoa {
    public static void main (String args[]) {
        Pessoa joao = new Pessoa();
        joao.nome = "João da Silva";
        joao.email = "josilva@gmail.com";
        System.out.println(joao.nome + ", " + joao.email);
    }
    
}

File: fund/webap/monopy/08-sessao-login-front-back/back-end/testes/TestarPessoa.py
from geral import *
from modelo.Pessoa import Pessoa

def run():
    print("TESTE DE PESSOA")
    
    p1 = Pessoa(nome = "João da Silva", email = "josilva@gmail.com", 
    telefone = "47 99012 3232")
    p2 = Pessoa(nome = "Maria Oliveira", email = "molive@gmail.com", 
        telefone = "47 98822 2531")        
    db.session.add(p1)
    db.session.add(p2)
    db.session.commit()
    print(p1, p2)